src/galaxy/SystemBody.cpp

```diff
@@ -262,6 +262,42 @@ bool SystemBody::IsScoopable() const
 	return false;
 }
 
+// all input units are in km
+double SystemBody::ComputeDensity(const double radius, const double atmosphereHeight, const double h, const double baricStep) const
```

Comment:
I'd strongly prefer this function be annotated with a description of both the math inside as well as its purpose.